server/src/main/java/com/project/yozmcafe/service/LikedCafeService.java

```diff
@@ -54,7 +53,9 @@ public List<LikedCafeResponse> findLikedCafesByMemberId(final String memberId) {
     }
 
     @Transactional
-    public void updateLike(final Member member, final long cafeId, final boolean isLiked) {
+    public void updateLike(final String memberId, final long cafeId, final boolean isLiked) {
+        final Member member = memberService.findMemberByIdOrElseThrow(memberId);
+
         final Cafe cafe = cafeRepository.findById(cafeId)
                 .orElseThrow(() -> new IllegalArgumentException("해당하는 카페가 존재하지 않습니다."));
 
```

Comment:
아니 이게 뭐야;; 이걸 이제 봤네요;;

```suggestion
    final Cafe cafe = cafeRepository.findById(cafeId)
            .orElseThrow(() -> new BadRequestException(NOT_EXISTED_CAFE));
```

요렇게 수정 부탁드려도 될까요~?